airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -777,6 +777,44 @@ definitions:
       type:
         type: string
         enum: [LegacyToPerPartitionStateMigration]
+  IncrementingCountCursor:
+    title: Incrementing Count Cursor
+    description: Cursor that allows for incremental sync according to a continuously increasing number
```

Comment:
nit: a `number` in json schema language can be a float. Should we be precise here and say it's integers only?